.pre-commit-config.yaml

```diff
@@ -1,27 +1,27 @@
----
-repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.2.0
-    hooks:
-      - id: check-added-large-files
-      - id: check-merge-conflict
-      - id: detect-private-key
-      - id: end-of-file-fixer
-      - id: fix-byte-order-marker
-      - id: no-commit-to-branch
-        args: [-b, main]
-      - id: trailing-whitespace
-  - repo: https://github.com/tconbeer/sqlfmt
-    rev: v0.20.0
-    hooks:
-      - id: sqlfmt
-        language_version: python
-        additional_dependencies: ['.[jinjafmt]']
-  - repo: local
-    hooks:
-      - id: yamlfix
-        name: yamlfix
-        types: [yaml]
-        language: system
-        entry: yamlfix --exclude ".kubernetes/**/*" --exclude ".venv/**/*" --exclude
-          "dbt_packages/**/*" --exclude "target/**/*" .
+---
+repos:

```

Comment:
sabe pq ele tá mostrando como se vc tivesse apagado e reescrito tudo de novo aqui? nao consigo identificar a diferença, mas to insegura que querbre alguma coisa

Reply:
Sinceramente meu pre-commit só corrigiu o próprio arquivo. Vou tentar voltar ele para a forma antiga e vê se contigo ignorar ele.

Reply:
Restaurei a versão dele antes do pre-commit modificar ele 🥳